addons/account/wizard/accrued_orders.py

```diff
@@ -110,6 +110,7 @@ def _get_aml_vals(order, balance, amount_currency, account_id, label=""):
                 'debit': balance if balance > 0 else 0.0,
                 'credit': balance * -1 if balance < 0 else 0.0,
                 'account_id': account_id,
+                'analytic_distribution': analytic_distribution,
```

Comment:
This will prevent computing the values if an automatic rule is applying to this line.
Is this wanted?